buffer/src/main/java/io/netty/buffer/AllocateBufferEvent.java

```diff
@@ -21,7 +21,7 @@
 
 @SuppressWarnings("Since15")
 @Label("Buffer Allocation")
-@Name("AllocateBufferEvent")
+@Name("io.netty.AllocateBufferEvent")
```

Comment:
I think you should change the name to @Name("io.netty.AllocateBuffer"). It's fine to keep the class name as AllocateBufferEvent. This is how we have done things in the JDK:
https://github.com/openjdk/jdk/tree/master/src/jdk.jfr/share/classes/jdk/jfr/events